packages/remix-dev/compiler/compileBrowser.ts

```diff
@@ -35,6 +35,14 @@ export type BrowserCompiler = {
   dispose: () => void;
 };
 
+let nodeBuiltins = Array.from(
+  new Set([
+    ...builtinModules,
+    // account for `node:`-prefixed built-ins like `"node:fs"`
+    ...builtinModules.map((x) => `node:${x}`),
+  ])
+);
```

Comment:
i'm sort of surprised that `builtInModules` doesn't already account for the `node:*` prefix tbh 🤷

Reply:
It does, but not sure since what version that's included tbh

https://github.com/nodejs/node/blob/762a3a8ad925d56a12b43e0e7f7c811e93097784/lib/internal/modules/cjs/loader.js#L311

Reply:
This was also introduced in v16.17 (see nodejs/node#43396), so we should account for values that can or can not have `node:`-prefix

A possible solution would be to map over all values, remove the `node:`-prefix, put that array in a Set so we have unique values, loop over the Set to add the `node:`-prefix again